src/Aquifer.API/Endpoints/Help/Aquifer-cms/Documents/Endpoint.cs

```diff
@@ -0,0 +1,40 @@
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.API.Endpoints.Help.Aquifer_cms.Documents;
+
+public class Endpoint(AquiferDbContext dbContext) : EndpointWithoutRequest<Response>
+{
+    public override void Configure()
+    {
+        Get("/help/aquifer-cms/documents");
+    }
+
+    public override async Task HandleAsync(CancellationToken ct)
+    {
+        var helpDocuments = await GetHelpDocumentsAsync(ct);
+        var response = new Response
+        {
+            Releases = helpDocuments.FindAll(x => x.Type == HelpDocumentType.Release),
+            HowTos = helpDocuments.FindAll(x => x.Type == HelpDocumentType.HowTo)
```

Comment:
We should be using `Where`. The next version of .NET, which comes out in a month, is going to have massive performance improvements to LINQ. Then to avoid calling `ToList()` I would make the properties `IEnumerable` instead of `List`.